python/tank/util/defaults_manager.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from tank_vendor import shotgun_authentication as sg_auth
+
+
+class CoreDefaultsManager(sg_auth.DefaultsManager):
+    """
+    This defaults manager implementation taps into the core's configuration to
+    provide a default host, proxy and user.
+    """
+
+    def is_host_fixed(self):
+        """
+        Returns if the host is fixed. Note that the defaults manager for a core
+        is always fixed, since a core works with the one and only host specified
+        in the configuration.
+        :returns: True
+        """
+        return True
+
+    def get_host(self):
+        """
+        Returns the host found in the core configuration.
```

Comment:
It used to. Last time around you suggested that I **shouldn't** mention shotgun.yml.

So, which is it? :D